### PR TITLE
Authors' list changed

### PR DESCRIPTION
**Pull request by marcj75**
Due to the change to the authors' list, the entry qin20a has been renamed peng20a:
- created folder peng20a and added files peng20a.pdf and supp.pdf
- removed folder qin20a
- removed file _posts/2020-06-03-qin20a.md
- created file _posts/2020-06-03-peng20a.md

### CODE REVIEW

**Comment by lawrennd** [comment]
Hi Marcello, 

The sump.pdf file needs to be named (I think peng20a-supp.pdf is right ... it's the name in the file at any rate).

I've liaised with Silvia to double check this (as adding an author name is unusual). I understand it's at the bequest of the authors and/or their institutions. So happy to approve when you've got that name mapped correctly.

Neil

**Comment by marcj75** [comment]
Dear Neil,

I have renamed the supplementary file.

Thanks,
Marcello

File: _posts/2020-06-03-peng20a.md [hunk 1]
```diff
@@ -23,12 +23,14 @@ lastpage: 2464
 page: 2455-2464
 order: 2455
 cycles: false
-bibtex_author: Qin, Yue and Peng, Jian and Zhou, Yuan
+bibtex_author: Peng, Jian and Qin, Yue and and Wei, Yadi Zhou, Yuan
```
Comment (lawrennd) [inline _posts/2020-06-03-peng20a.md]:
```suggestion
bibtex_author: Peng, Jian and Qin, Yue and Wei, Yadi and Zhou, Yuan
```

**Review by lawrennd — APPROVED** [review]

**Comment by lawrennd** [comment]
Thanks Marcello, had to make a minor adjustment, but now merging, can you check that the links work post-merge (it should take a few minutes to recompile). Thanks!

**Pull request merged**